Tests/QuickTests/QuickTests/Helpers/QuickSpec+QuickSpec_MethodList.m

```diff
@@ -4,27 +4,19 @@
 
 @implementation QuickSpec (QuickSpec_MethodList)
 
-/**
- *  This method will instantiate an instance of the class on which it is called,
- *  returning a list of selector names for it.
- *
- *  @warning Only intended to be used in test assertions!
```

Comment:
As the message said, the function should not be defined in `Quick-macOS` and so on.